src/styles/components/pods-view.less

```diff
@@ -0,0 +1,125 @@
+.pod-instances-table {
+
+  tbody {
+    line-height: 1.8;
```

Comment:
Let's base this off of an existing variable. Since the table line-height is set to `null`:
https://github.com/mesosphere/canvas/blob/master/styles/variables/variables-tables.less#L109

let's instead use this body text line-height:
https://github.com/mesosphere/canvas/blob/master/styles/variables/variables-typography.less#L58